.codegen/changelog_config.yml

```diff
@@ -0,0 +1,15 @@
+change_types:
+  - message: New Features and Improvements
+    tag: "[Feature]"
+  - message: Bug Fixes
+    tag: "[Fix]"
+  - message: Documentation
+    tag: "[Doc]" 
+  - message: Internal Changes
+    tag: "[Internal]"
+  - message: Dependency Updates
```

Comment:
probably would come under Internal Changes?

Reply:
Discussed offline: The previous changelog has a separate subsection for dependency updates. Leaving it to minimize changes. We can always change it later on.